content/blog/metafacture-core-5.3.1/index.md

```diff
@@ -0,0 +1,238 @@
+a nested hierachy of only two. Also, the structure was kind of broken e.g. for JSON.
+
+Using the following FLUX:
+
+```
+"testArray.json"
+| open-file
+| as-records
+| decode-json
+| morph("all.xml")
+| encode-json(prettyPrinting="true")
+| write("stdout");
+```
+
+and the morph:
+```
+<?xml version="1.0" encoding="UTF-8"?>
+<metamorph xmlns="http://www.culturegraph.org/metamorph" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	version="1">
+	<rules>
+<data source="_elseNested"/>
+</rules>
+</metamorph>
+```
+
+with this JSON input:
+```JSON
+{
+    "author": [
+        {
+            "@type": "Person",
+            "name": "Katja Königstein-Lüdersdorff"
+        },
+        {
+            "@type": "Person",
+            "name": "Corinna Peters"
+        },
+        {
+            "@type": "Person",
+            "name": "Oleg Tjulenev"
+        },
+        {
+            "@type": "Person",
+            "name": "Claudia Vogeler"
+        }
+    ]
+}
+```
+
+
+resulted in:
+```JSON
+{
+  "1" : {
+    "@type" : "Person",
+    "name" : "Katja Königstein-Lüdersdorff"
+  },
+  "2" : {
+    "@type" : "Person",
+    "name" : "Corinna Peters"
+  },
+  "3" : {
+    "@type" : "Person",
+    "name" : "Oleg Tjulenev"
+  },
+  "4" : {
+    "@type" : "Person",
+    "name" : "Claudia Vogeler"
+  }
+}
+```
+
+With the new release the output is:
+
+```JSON
+{
+  "author" : [ {
+    "@type" : "Person",
+    "name" : "Katja Königstein-Lüdersdorff"
+  }, {
+    "@type" : "Person",
+    "name" : "Corinna Peters"
+  }, {
+    "@type" : "Person",
+    "name" : "Oleg Tjulenev"
+  }, {
+    "@type" : "Person",
+    "name" : "Claudia Vogeler"
+  } ]
+}
+```
+
+### Morph function "setreplace" with filemaps
+You can now use the morph function `setreplace` with externals file maps. These maps
+are lists of tabulator separated values (tsv): the first value is, when matched,
+substituted by the second value. You can also leave out a second value resulting in the
+removal of the matched value. Use it like this:
+
+```
+       ...
+       <rules>
+          <data source='fieldNameWhereValuesShallBeSubstituded'>
+            <setreplace map='mapname' />
+          </data>
+        </rules>
+        <maps>
+          <filemap name='mapname' files='org/metafacture/metamorph/maps/file-map-test.txt' />
+        </maps>
+        ...
+```
+
+## Outlook
+We are working on a [Catmandu like fix language](https://librecat.org/Catmandu/#fix-language) which can be used instead of the `morph` script.
+Also, there will be a [playground](https://metafacture.org/playground), realized as
+a web app, to play around with data and transformation rules and see the outcome
+immediately. You will be offered to load predefined examples.
+The `playground` comes with the capability to share examples or whole complex
+workflows, with the vision to enable this as a web API for processing data without
+even installing metafacture.
```

Comment:
What is your concrete suggestion: 
a) just remove this section or 
b) insert a link to the (to be written) outlook blog post? 
I thought it would be nice, if any reader comes so far to read this section, to plant into the readers mind that further posts might be also worth to read.

Reply:
Then keep it like this. But make a hint on the new developement in metafacture.

Reply:
Not sure what you mean - this "Outlook" is intended to exactly hint this new development. What do you feel is missing?

Reply:
As discussed added a "watch out" sentence,